src/Compilers/CSharp/Portable/Symbols/FunctionPointerTypeSymbol.cs

```diff
@@ -0,0 +1,264 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.Reflection;
+using Microsoft.Cci;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.PooledObjects;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.CSharp.Symbols
+{
+    internal class FunctionPointerTypeSymbol : TypeSymbol
```

Comment:
>FunctionPointerTypeSymbol [](start = 19, length = 25)

Both classes should override and implement GetUseSiteDiagnostics

Reply:
See what else is being overridden by PointerTypeSymbol, I would expect all of that overridden here too. Equality, GetHashCode, etc.

---
In reply to: [359072058](https://github.com/dotnet/roslyn/pull/40431#discussion_r359072058) [](ancestors = 359072058)